Client/mods/deathmatch/logic/CClientGame.cpp

```diff
@@ -6748,47 +6748,49 @@ bool CClientGame::TriggerBrowserRequestResultEvent(const std::unordered_set<SStr
     return GetRootEntity()->CallEvent("onClientBrowserWhitelistChange", Arguments, false);
 }
 
-void CClientGame::RestreamModel(unsigned short usModel)
+bool CClientGame::RestreamModel(const std::uint16_t model)
```

Comment:
What’s the point of making this function return a bool if it always returns true anyway?
void would be more appropriate here.